unreal_asset/src/asset/name_map.rs

```diff
@@ -79,14 +79,16 @@ impl NameMap {
     }
 
     /// Get a name reference by an FName map index
-    pub fn get_name_reference(&self, index: i32) -> String {
+    pub fn get_name_reference(&self, index: i32) -> &str {
+        // to avoid the panic this could return an option instead
+
         if index < 0 {
-            return (-index).to_string(); // is this right even?
+            return "too low";
```

Comment:
maybe make it more descriptive like "ERROR: name reference index too low"?

Reply:
sure can do